backend/bitboxbase/bitboxbase.go

```diff
@@ -143,8 +145,8 @@ func (base *BitBoxBase) GetUpdaterInstance() *updater.Updater {
 }
 
 //BlockInfo returns the received blockinfo packet from the updater
-func (base *BitBoxBase) BlockInfo() string {
-	return base.GetUpdaterInstance().BlockInfo()
+func (base *BitBoxBase) BlockInfo() interface{} {
```

Comment:
also rename to MiddlewareInfo then?

return type should ideally be a concrete type (struct, ...?) so it's clear what is expected. then it can also be validated.

Reply:
Can I do this in the next pull request, where I'll clean up the websocket code?